Firebase/Messaging/FIRMessaging.m

```diff
@@ -771,6 +777,7 @@ - (void)subscribeToTopic:(NSString *)topic
   if (completion) {
     completion([NSError fcm_errorWithCode:FIRMessagingErrorInvalidTopicName userInfo:nil]);
   }
+  }];
```

Comment:
nit: lines above this are mis-indented (or GitHub UI is weird?)

Reply:
Done. Yeah it's mis-indented.